assets/src/blocks/ShareButtons/ShareButtonsBlock.js

```diff
@@ -80,12 +80,13 @@ export const registerBlock = () => {
     },
     attributes,
     edit: ShareButtonsEditor,
-    save: ({ attributes }) => {
-      if(!attributes) {
+    // eslint-disable-next-line no-shadow
```

Comment:
I think that a better solution here would be to remove the previous `attributes` const (since it's only used once) and write them directly in the attributes, then here this disable is not needed :)

Reply:
Done.